Detectors/Base/include/DetectorsBase/DCAFitter.h

```diff
@@ -227,7 +227,7 @@ class DCAFitter
     setUseAbsDCA(false); // by default use weighted DCA definition (much slower)
   }
 
-  ///< number of validated V0 candidates (at most 2 are possible)
+  ///< number of validated FV0 candidates (at most 2 are possible)
```

Comment:
I've already commented on this in the previous PR https://github.com/AliceO2Group/AliceO2/pull/2255#discussion_r307315630, these are secondary vertices V0s, please revert this file